src/modules/fundingManager/bondingCurveFundingManager/formula/BancorFormula.sol

```diff
@@ -0,0 +1,541 @@
+pragma solidity 0.8.19;
+
+import {IBancorFormula} from 
+    'src/modules/fundingManager/bondingCurveFundingManager/formula/IBancorFormula.sol';
+import {SafeMath} from 
+    'src/modules/fundingManager/bondingCurveFundingManager/formula/SafeMath.sol';
+import {Utils} from
+    'src/modules/fundingManager/bondingCurveFundingManager/formula/Utils.sol';
+
+/// @title  Aragon Bancor Formula
+/// @author https://github.com/AragonBlack/fundraising/blob/master/apps/bancor-formula/contracts/BancorFormula.sol
+/// @notice The sole modification applied to these contracts involves the alteration of the Solidity
```

Comment:
I see the point you are making here, it could be that some behaviour was intended that now potentially changed because of the default SafeMath.

This comes back to earlier discussions we had internally about trying to touch the formula contract as little as possible to keep the integrity, because we are not testing it within our repo. Any thoughts here @FHieser @marvinkruse

Reply:
Also the change in `import` statements is mentioned already but will reformulate it for clarity

Reply:
Tagging @marvinkruse here for looking into compiling different Solidity Version with Foundry